README.md

```diff
@@ -32,14 +36,24 @@ unzip master.zip
 
 ```bash
 cd hello_looper
-looper run --looper-config .looper.yaml
+
+looper run --looper-config looper_config.yaml
```

Comment:
I think the previous version was better. we generally expect people to use `.looper.yaml` as the default config file name. why change it from the default?